website/highlights/2020-07-10-add-reduce-transform.md

```diff
@@ -0,0 +1,52 @@
+---
+last_modified_on: "2020-07-15"
+$schema: "/.meta/.schemas/highlights.json"
+title: "New Reduce transform"
+description: "More flexibility around grouping events."
```

Comment:
What do you think about changing to this one of:

1. Stripe's canonical log lines in Vector
2. The new `reduce` transform reduces your logs